tests/unittests/tests-hashes/tests-hashes.h

```diff
@@ -107,12 +107,19 @@ Test *tests_hashes_sha256_hmac_tests(void);
  */
 Test *tests_hashes_sha256_chain_tests(void);
 
-  /**
- * @brief   Generates tests for hashes/sha3.h
+/**
+ * @brief   Generates tests for hashes/sha512.h - generic hmac
+ *
+ * @return  embUnit tests if successful, NULL if not.
+ */
+Test *tests_hashes_sha512_hmac_tests(void);
```

Comment:
We have all the other PSA crypto API tests below `tests/sys/psa_crypto*`. To not bloat the `unittests` binary too much, I would propose you move the tests to the already existing `psa_crypto_mac` test.

Reply:
Good point, I moved the tests to the existing `psa_crypto_mac` test from `unittests`.